index.html

```diff
@@ -1756,15 +1786,33 @@ <h5>Default Reading Order</h5>
 							</tbody>
 						</table>
 
+						<p>Each element of the <code>readingOrder</code> property MUST be expressed either as:</p>
+
+						<ul>
+							<li>a [[!json]] <a href="https://tools.ietf.org/html/rfc4627#section-2.5">string</a>
+								representing the <a>URL</a> of the resource; or</li>
+							<li>an instance of a <a href="#publication-link-def"><code>LinkedResource</code></a>
+								object.</li>
+						</ul>
+
+						<p>A single string value represents an instance of a <code>LinkedResource</code> object whose
+								<code>url</code> property is the string's text.</p>
+
```

Comment:
I'm not sure what to say about this one. If it was a MUST, then it would definitely make sense, but we could just be adding confusion by adding a media type when we only recommend omitting for HTML.

It might be better to just leave it to the user agent to make sense of the resource by its extension, for example.

Reply:
O.k., I agree.